src/components/event/EventList.tsx

```diff
@@ -126,6 +126,10 @@ function EventTreeListBase(props: Props) {
 		}
 	}, [scrolledIndex]);
 
+	React.useEffect(() => {
+		initialItemCount.current = nodes.length;
```

Comment:
I'm not sure why do we need this?

This property applies only on mount and it's default value declared in initialItemCount ref should be enough